### PR TITLE
Update Issue template for SPM

### PR DESCRIPTION
**Pull request by scannillo**
### Summary of changes

- Update issue template for SPM

### CODE REVIEW

**Review by sestevens — APPROVED** [review]
🎉

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ about: Create a report to help us improve
 <!-- Only open an issue here if you think you've found an issue with our SDK. If you need help troubleshooting your integration, reach out to Braintree Support at https://help.braintreepayments.com. -->
 
 **Integration Details (please complete the following information):**
- - SDK/Library Version: [e.g. 4.33.0]
+ - SDK/Library Version: [e.g. 5.0.0]
```
Comment (sestevens) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]: If we want, this could probably just say `Braintree SDK` version.

Comment (sestevens) [reply .github/ISSUE_TEMPLATE/bug_report.md:9]: Instead of `SDK/Library Version`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ about: Create a report to help us improve
...
  - Environment: [e.g. Sandbox or Production]
- - iOS Version: [e.g. iOS 13.0]
- - Device [e.g. iPhone 6s]
- - Integration type & version [e.g. CocoaPods 1.1.1, Carthage 0.18.1]
+ - iOS Version: [e.g. iOS 14.0]
+ - Device [e.g. iPhone 12]
+ - Integration type & version [e.g. CocoaPods 1.1.1, Carthage 0.18.1, Swift Package Manager 5.2.0]
```
Comment (sestevens) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
 - Device: [e.g. iPhone 12]
 - Integration type & version: [e.g. CocoaPods 1.1.1, Carthage 0.18.1, Swift Package Manager 5.2.0]
```

**Review by demerino — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (scannillo) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]:
```suggestion
 - Braintree SDK Version: [e.g. 5.0.0]
```

**Pull request merged**